sensors/ov3660.c

```diff
@@ -18,9 +17,11 @@
 #include "freertos/task.h"
 
 #if defined(ARDUINO_ARCH_ESP32) && defined(CONFIG_ARDUHAL_ESP_LOG)
+#include "Arduino.h"
 #include "esp32-hal-log.h"
 #else
 #include "esp_log.h"
+#define delay(x)
```

Comment:
is delay used in this code? should change to vTaskDelay :)

Reply:
Is it used in ARDUINO_ARCH_ESP32 ?

Reply:
there should not be any calls to Arduino's `delay` function in the driver. It is equal of vTaskDelay per millisecond.

Reply:
and to explain, that check for arduino is so if it is detected, to forward the ESP_LOG calls to Arduino's logging facility. Else errors and warnings might be left hidden